src/sass_types/factory.cpp

```diff
@@ -45,26 +43,27 @@ namespace SassTypes
     }
   }
 
-  void Factory::initExports(Handle<Object> exports) {
-    Local<Object> types = NanNew<Object>();
-    exports->Set(NanNew("types"), types);
+  void Factory::initExports(v8::Local<v8::Object> exports) {
+    Nan::EscapableHandleScope scope;
```

Comment:
> > -  void Factory::initExports(Handle<Object> exports) {
> > -    Local<Object> types = NanNew<Object>();
> > -    exports->Set(NanNew("types"), types);
> > +  void Factory::initExports(v8::Local<v8::Object> exports) {
> > +    Nan::EscapableHandleScope scope;
> 
> Its return type is void, a normal HandleScope will suffice.

Re-read "V8 Embedders Guide" which says "you cannot return a local handle
directly from a function that declares a handle scope". 

Thank you very much!